# 모드
string mode  # 사용할 좌표계: "ENU" 또는 "UTM"

# ENU 좌표계
float64 enu_east
float64 enu_north
float64 enu_up
float64 yaw

# WGS84 좌표계
float64 utm_x
float64 utm_y
float64 heading

